### PR TITLE
Align stable with canary

### PR DESCRIPTION
**Pull request by 2can**

### CODE REVIEW

File: versioned_docs/version-3.x/developer/app-store/apps/avatax/configuration.mdx [hunk 1]
```diff
@@ -132,6 +132,8 @@ When the AvaTax app is handling checkout or order calculate taxes webhooks it is
 
 For order confirmed webhook, the AvaTax app is using [price reduction discounts flow](https://developer.avalara.com/erp-integration-guide/sales-tax-badge/transactions/discounts-and-overrides/discounting-a-transaction/). It means that Saleor is responsible for calculating discounts and AvaTax app is sending discounted prices to Avalara for tax calculation.
 
+For more information about how discounts work in Saleor see discounts [guide](../../../../developer/discounts/overview) from our docs.
```
Comment (aniav) [inline versioned_docs/version-3.x/developer/app-store/apps/avatax/configuration.mdx:135]:
```suggestion
For more information about how discounts work in Saleor see discounts [guide](developer/discounts/overview.mdx) from our docs.
```

**Review by aniav — APPROVED** [review]
One nonblocking nitpick, aside from that it's good to go 👍

File: versioned_docs/version-3.x/developer/app-store/apps/avatax/configuration.mdx (hunk 1 above)
Comment (2can) [reply versioned_docs/version-3.x/developer/app-store/apps/avatax/configuration.mdx]: @aniav done, but that shouldn't be in the scope of this PR, as here the idea is to align canary and stable meaning they contain the same content, the requested change affects both so it's not an alignment :wink:

Comment (aniav) [reply versioned_docs/version-3.x/developer/app-store/apps/avatax/configuration.mdx]: Yup, that's why I said it's nonblocking 😂

**Pull request merged**